be/src/storage/lake/persistent_index_memtable.cpp

```diff
@@ -176,7 +176,10 @@ Status PersistentIndexMemtable::get(const Slice* keys, IndexValue* values, const
 }
 
 size_t PersistentIndexMemtable::memory_usage() const {
-    return _keys_size + _map.size() * sizeof(IndexValueWithVer);
+    // _keys_heap_size is the memory usage of std::string which are heap allocated.
+    // _map.bytes_used() is the memory usage of the btree.
+    // The total memory usage is the sum of these two.
+    return _keys_heap_size + _map.bytes_used();
```

Comment:
wondering if the `_keys_heap_size` is the major deviation or _map.bytes_used() will be good enough?

Reply:
@kevincai `_keys_heap_size` took a very obvious piece, almost 50%